brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-1971 - Slider Block General Settings

### Description
This change has all the General settings related to the Slider block. Following are the options added in this change, 

-  Pause On ( Hover, click, none )
- Autoplay
    - Autoplay speed
- Infinite Loop
- Transition Effect ( Slide, Fade, Flip )
- Transition Speed  
- Show Arrows & Dots  ( Arrow, Dots, Both )

### Types of changes
New feature (non-breaking change which adds functionality)


### How has this been tested?
- Add a slider Block 
- Click on Slider -> General Settings 
- Check all settings, if the live preview is working for all the options
- Save changes and check if settings are reflected on the front end. 

Note: **Pause on** option preview will only work on the Front end. 


### Checklist:
